docs/getting-started.md

```diff
@@ -176,6 +176,33 @@ Create two new classes `example.ShowsDataFetcher` and `Show` and add the followi
 
 That's all the code needed, the application is ready to be tested!
 
+## Using @InputArgument
+You may have noticed the use of `@InputArgument` to extract the input arguments from your data fetching environment.
+This should work for most scenarios as shown above, such as `String`, `Integer`, input objects, custom scalars and lists.
+If you have a list of input object types, however, you will also need to specify the collection type for proper deserialization as shown below:
+
+=== "Java"
+    ```java
+        @DgsData(parentType = DgsConstants.MUTATION.TYPE_NAME, field = DgsConstants.MUTATION.AddReview)
```

Comment:
Maybe split in two sections. It's confusing that the text is about lists, but the example starts with something that isn't a list.

Reply:
Separated it out as suggested.